src/Pako.js

```diff
@@ -1,16 +1,20 @@
 "use strict";
-var pako = require('pako');
+import pako from 'pako';
 
-exports.deflateImpl = function(options) {
-  return function(bytes) {
-    return function() {
-      return pako.deflate(bytes, options).buffer;
-    };
-  };
-};
+export const hello = () => console.log("hello");
```

Comment:
I guess this could removed?

Reply:
That's embarrassing, I'm still getting used to ES6 module syntax, I must have pasted that in as a reference and forgotten about it. Thanks.